app/src/main/java/eu/kanade/domain/track/service/TrackPreferences.kt

```diff
@@ -36,6 +36,8 @@ class TrackPreferences(
 
     fun anilistScoreType() = preferenceStore.getString("anilist_score_type", Anilist.POINT_10)
 
+    fun privateTracking() = preferenceStore.getBoolean("pref_private_tracking", false)
```

Comment:
It's already opt-in we don't another extra toggle for this

Reply:
Users can already choose if they want to track privately or not. No need for an additional toggle on top of that